### PR TITLE
Move EnvCompare::CLI out to it's own file

### PR DESCRIPTION
**Pull request by jaydorsey**
- Cleans up the behavior by putting it where one would expect to find it
- Adjusts the theme path to compensate

### CODE REVIEW

**Comment by smridge** [comment]
We should move the related spec to `spec/env_compare/cli_spec.rb`.

**Comment by smridge** [comment]
Should we name Class `Cli` vs `CLI` to avoid confusion that may lead to think it's a constant?

File: lib/env_compare/cli.rb [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    end
+
+    def heroku_results(envs = nil)
+      @heroku_results ||=
+        envs.each_with_object({}) do |env, obj|
+          obj[env] = {}
+          lines = `heroku config -a #{env}`.split("\n").drop(1)
+
+          lines.each do |line|
+            key, _, value = line.partition(':')
```
Comment (smridge) [inline lib/env_compare/cli.rb:65]: I think we can do `key, value = line.split(':')` and avoid the unnecessary `_` since `strip` is being used regardless?

Comment (jaydorsey) [reply lib/env_compare/cli.rb:65]: This was a copy/paste. This goes away in my next PR

Comment (smridge) [reply lib/env_compare/cli.rb:65]: Sweet!

**Comment by jaydorsey** [comment]
> Should we name Class `Cli` vs `CLI` to avoid confusion that may lead to think it's a constant?

I'm kind of torn on this. I think `CLI` reads better because it's an acronym, so every letter gets capitalized, like `API`, or `XML`. I've definitely seen code do `Xml` before, and even `Json` and `Api`...

Ruby constants only need to start with an upper case; it's really just convention that we see them all uppercase. I'm not even sure why it's _almost_ always the case they're ALL upper cased. I can't think of a single constant I've seen that hasn't been all upper case 🤔 

image

I've definitely seen the convention that "constants are all caps" but haven't really seen a preference for "all caps are only for constants"

I definitely prefer `CLI` but it's also not that big a deal so I'll change it

**Pull request merged**